templates/plugin/kafkaoutput.toml.erb

```diff
@@ -0,0 +1,44 @@
+# This file is controlled via puppet.
+[<%= @plugin_name -%>]
+type = "KafkaOutput"
+<%= scope.function_template(['heka/plugin/_output.toml.erb']) %>
```

Comment:
This should be move to the end of the file. Because output common parameters have also buffering parameters, which is a sub-section, it's more aesthetic. And I'm not sure how heka behaves if after a subsection you return to the main section